backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/concept.html.ftl

```diff
@@ -1,54 +1,58 @@
-		<@layout.kc k="Table">
-		<a href="/admin-ui/datasets/${c.dataset.id}/tables/${connector.table.id}">${connector.table.name}</a>
-        </@layout.kc>
-		<@layout.kc k="Filters">
-			<ul>
-			<#list connector.collectAllFilters()?sort_by("name") as filter>
-				<li>
-					<@layout.kid k="ID" v=filter.id/>
-					<@layout.kv k="Label" v=filter.label/>
-					<@layout.kv k="Type" v=filter.class.simpleName/>
-					<@layout.kv k="Columns" v=filter.requiredColumns?sort_by("name")?join(', ')/>
-				</li>
-			</#list>
-			</ul>
-		</@layout.kc>
-		<@layout.kc k="Selects">
-			<ul>
-			<#list connector.selects?sort_by("name") as select>
-				<li>
-					<@layout.kid k="ID" v=select.id/>
-					<@layout.kv k="Label" v=select.label/>
-					<@layout.kv k="Type" v=select.class.simpleName/>
-					<@layout.kv k="Columns" v=select.requiredColumns?sort_by("name")?join(', ')/>
-				</li>
-			</#list>
-			</ul>
-		</@layout.kc>
-	</#list>
-	</@layout.kc>
+  <@breadcrumbs.breadcrumbs
+    labels=["Datasets", c.dataset.label, "Concepts", c.label]
+    links=[
+      "/admin-ui/datasets",
+      "/admin-ui/datasets/${c.dataset.id}",
+      "/admin-ui/datasets/${c.dataset.id}#Concepts"
+    ]
+  />
+  <@infoCard.infoCard
+    class="d-inline-flex"
+    title="Concept ${c.label}"
+    labels=["ID", "Label", "Type", "Structure Parent", "Elements"]
+    values=[c.id, c.label, c.class.simpleName, c.structureParent!"", c.countElements()?string.number]
+  />
+
+  <@accordion.accordionGroup class="mt-3">
+    <@accordion.accordion summary="Selects" infoText="${c.selects?size} entries">
+      <@table.table
+        columns=["id", "label", "simpleName", "description"]
+        items=c.selects?sort_by("name")?map(x -> {"id": x.id, "label": x.label, "simpleName": x.class.simpleName, "description": x.description!""})
+      />
+    </@accordion.accordion>
+    <@accordion.accordion summary="Connectors" infoText="${c.connectors?size} entries">
+	    <#list c.connectors as connector>
+        <div class="d-flex flex-row align-items-start my-3" style="gap: 0.5rem;">
+          <@infoCard.infoCard
+            labels=["ID", "Label", "Validity Dates", "Table"]
+            values=[connector.id, connector.label, connector.validityDates?join(', '), connector.table.name]
+            gridTemplateColumns="auto"
+            style="flex-basis: 22rem; flex-shrink: 0;"
+            links={"Table": "/admin-ui/datasets/${c.dataset.id}/tables/${connector.table.id}"}
+          />
+          <@accordion.accordionGroup style="flex-grow: 1; margin-top: 0 !important;">
+            <@accordion.accordion summary="Filters" infoText="${connector.collectAllFilters()?size} entries">
+              <@table.table
+                columns=["id", "label", "simpleName", "requiredColumns"]
+                items=connector.collectAllFilters()?sort_by("name")?map(x -> {"id": x.id, "label": x.label, "simpleName": x.class.simpleName, "requiredColumns": x.requiredColumns?sort_by("name")?join(', ')})
```

Comment:
😭

Reply:
Bist du mit dem inline Code nicht zufrieden?

Reply:
Guess I was just surprised, because the code we're writing is usually supposed to be maintained by someone over a longer period, but maybe this project is more about: build quickly, change almost never?

Since I'm not really involved, I also don't have context on
- why we're using `ftl` templates
- whether it allows better formatting 
- whether it allows extracting some logic

so please feel free to ignore this comment. I just stumbled over this while browsing the PRs a bit :)

Reply:
I see your point.

The usage of FTL is both historic in conquery but also the basic approach in dropwizard for templating.

Regarding the formatting and extracting logic, it's my fault, I overlooked it and will provide a better review.